content/en/contribute/medic/onboarding/product-team.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Product Team"
+linkTitle: "Product Team"
+weight: 3
+description: >
+  Introduction to Product Team 
+---
+
+The goal of this page is to convey a general “lay of the land” so someone starting can see a lot of what’s out there related to Product Team without having to be surprised each day as new things pop up.
+
+## About Product Team
+The Product Team manages the entire software development life-cycle to understand problems, capture requirements, design and build modular software systems, and document everything along the way.
+We achieve this by working with CHT partners and health workers to design, build, and maintain the Community Health Toolkit and its open source tools.
+
+## Composition
+The Product team comprises a community team and 4 Focused Working Groups, each focusing on outcomes to serve different people.
```

Comment:
```suggestion
The Product Team comprises a Community Team and 4 Focused Working Groups, each focusing on outcomes to serve different people.
```

Reply:
For the sake of consistency with the paragraphs above.

Reply:
Thank you, fixed!